docs/development/README.adoc

```diff
@@ -0,0 +1,73 @@
+:toc: macro
+
+= T developer documentation
+
+toc::[]
+
+== <<local-t-network.adoc,Setting up local T network>>
+
+For development purposes, it is convenient to set up a local Ethereum client with
+a group of connected peers. A note-taken walkthrough covers the entire process
+including setting up a local Ethereum client, deploying T contracts and
+configuring T peers.
+
+== Quick installation
+
+To quickly install and start a client use the following scripts.
+
+=== Install scripts
+
+The `+install.sh+` script will:
+
+* install yarn dependencies
+* unlock accounts
+* build contracts
+* deploy contracts
+* build client
+
+The `+initialize.sh+` script will:
+
+* mint & approve `T` tokens
+* stake `T` tokens
+* increase authorization for `RandomBeacon`
+* register operator
+
+Please refer to the scripts `help` for available command arguments and environment variables. `./install.sh --help` or `./initialize.sh --help`. Accounts unlocking and staking can be run outside of these scripts as standalone hardhat tasks.
+
+To start the installation:
+
+* run `./scripts/install.sh`. You need to run it only once.
+
+* run `./scripts/initialize.sh --stake-owner <address>` for the default setup for each client. Default setup assumes that the staking provider, operator, beneficiary and an authorizer are all have the same addresses as the stake owner. Only the `stake-owner` has to be unique for the default setup. The default stake amount is `1,000,000 T` tokens. The default authorization amount is a min. authorization amount \n. 
```

Comment:
Was it expected that `\n` breaks a line? Because if so, it does not:

![Uploading image.png…]()

Reply:
[8c6255f](https://github.com/keep-network/keep-core/pull/3050/commits/8c6255f0b1af65fcd691c19486371ca610f9f280)